scripts/scripts/testing/automations/batch/georgia.py

```diff
@@ -0,0 +1,23 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import datetime
+
+url = 'https://stopcov.ge'
+req = requests.get(url)
+soup = BeautifulSoup(req.text, 'html.parser')
+
+daily = data.find_all('span',attrs={'class':'quantity-numver'}[6])
+daily = daily.text.replace(' ','')
+
+date = str(date.today())

```

Comment:
Avoid using `date` as the variable name, proposal: use `date_str`.

Forgot to import the library module `date`: 
```
from datetime import date
```

Reply:
I currently collect cumulative testing data for Georgia manually which works well for now